timeless/restaurants/tables/views.py

```diff
@@ -8,6 +8,11 @@
 @todo #112:30min Modify this file and also all other views.py files so the
  endpoints are registered inside timeless/__init__.py file in register_api
  method. Just like it was done for companies management views.
+@todo #309:30min Add mock authentication to TableListView. Currently
+ TabeListView test is not working because it requires an authenticated user
+ with permissions to show list tables. it is currently showing login screen.
+ Add an mocked authenticated user so we can run the test, the remove skip
+ annotation from it.
```

Comment:
@paulodamaso Can you add a test to check that indeed ListView returns the login page if there is no authenticated user?

Reply:
Done!